app/controllers/parks_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
   end
 
   def show
-    @park = Park.find(params[:id])
+    park
```

Comment:
Remove this, you should use a callback to run this for the appropriate actions `before_action :park, only: [:show,  :edit, :update, :destroy]`